.github/workflows/ci.yml

```diff
@@ -589,6 +635,12 @@ jobs:
         run: |
           docker push litentry/litentry-parachain
 
+      # only push the docker image if we rebuilt it
```

Comment:
Just one minor issue. Maybe it's better to rename this step. 
`image -> image (tee-prod)`
But this is not a big issue.